init-py3.py

```diff
@@ -0,0 +1,31 @@
+from __future__ import division
+from sympy import *
+
+from sympy.plotting import plot_parametric, plot3d
+
+# Output of the last expression
+_ = None
+
+x, y, z, t = symbols('x,y,z,t')
+k, m, n = symbols('k,m,n', integer=True)
+f, g, h = symbols('f,g,h', cls=Function)
+
+# lambda is a reserved word, pi is a built-in symbol
+
+_greek = 'alpha beta gamma delta epsilon zeta eta ' \
```

Comment:
Some of these are also SymPy names (namely `gamma` and `beta`).

Reply:
Good point. I will append an underscore to the symbol names and tell TeXmacs to do that too.

Reply:
That way pi will be back as pi_ :)